### PR TITLE
Routing, Page data & front-end menus 

### PR DESCRIPTION
**Pull request by jakxnz**
* https://trello.com/c/WK1LFinJ

NOTE! This PR includes a `composer update` and a `composer install` of October CMS plugins. That means there's a large number of files updated in the PR that we don't manage, and don't need to be reviewed.

### CODE REVIEW

File: plugins/underflip/resorts/models/Resort.php [hunk 1]
```diff
@@ -66,7 +66,9 @@ class Resort extends Model
      */
     public function getUrlAttribute()
     {
-        return sprintf('resort/%s', $this->url_segment);
+
```
Comment (Krank3n) [inline plugins/underflip/resorts/models/Resort.php:69]: where these line spaces added for any reason?

File: plugins/underflip/resorts/updates/seed_settings.php [hunk 2]
```diff
@@ -12,8 +12,10 @@ public function run()
         /** @var Settings $settings */
         $settings = Settings::instance();
 
-        $settings->copyright_message =
-            '[Shredindex](https://shredindex.com/) © 2021 Developed by [Underflip](https://github.com/underflip/)';
+        $settings->copyright_message = sprintf(
+            '[Shredindex](https://shredindex.com/) © %s Developed by [Underflip](https://github.com/underflip/)',
+            date('Y')
```
Comment (Krank3n) [inline plugins/underflip/resorts/updates/seed_settings.php:17]: nice

File: themes/underflip/meta/static-pages.yaml [hunk 3]
```diff
@@ -0,0 +1 @@
+static-pages: {  }
```
Comment (Krank3n) [inline themes/underflip/meta/static-pages.yaml:1]: whats this gonna be?

**Review by Krank3n — APPROVED** [review]
Just left one comment with the spacing I was unsure about... if you could take a look before merging it would be great :)

File: plugins/underflip/resorts/models/Resort.php (hunk 1 above)
Comment (jakxnz) [reply plugins/underflip/resorts/models/Resort.php:69]: Nope, I'll remove them

File: themes/underflip/meta/static-pages.yaml (hunk 3 above)
Comment (jakxnz) [reply themes/underflip/meta/static-pages.yaml:1]: Nothing. Leftover from a different approach. I'll remove it

**Pull request merged**